tests/unit_tests/test_helpers.py

```diff
@@ -41,41 +41,34 @@ def setup(self):
         logger.warning("Right before table creation")
         self.connection.create_table(
             TableName='shortlinks_test',
-            AttributeDefinitions=[
-                {
-                    'AttributeName': 'url', 'AttributeType': 'S'
-                }, {
-                    'AttributeName': 'shortlinks_id', 'AttributeType': 'S'
-                }
-            ],
-            KeySchema=[
-                {
-                    'AttributeName': 'shortlinks_id', 'KeyType': 'HASH'
-                }, {
+            AttributeDefinitions=[{
+                'AttributeName': 'url', 'AttributeType': 'S'
+            }, {
+                'AttributeName': 'shortlinks_id', 'AttributeType': 'S'
+            }],
+            KeySchema=[{
+                'AttributeName': 'shortlinks_id', 'KeyType': 'HASH'
+            }, {
+                'AttributeName': 'url', 'KeyType': 'HASH'
+            }],
+            LocalSecondaryIndexes=[{
+                'IndexName': 'UrlIndex',
+                'KeySchema': [{
                     'AttributeName': 'url', 'KeyType': 'HASH'
+                }],
+                'Projection': {
+                    'ProjectionType': 'INCLUDE', 'NonKeyAttributes': ['shortlinks_id']
                 }
-            ],
-            LocalSecondaryIndexes=[
-                {
-                    'IndexName': 'UrlIndex',
-                    'KeySchema': [{
-                        'AttributeName': 'url', 'KeyType': 'HASH'
-                    }],
-                    'Projection':
-                        {
-                            'ProjectionType': 'INCLUDE', 'NonKeyAttributes': ['shortlinks_id']
-                        }
-                },
-                {
-                    'IndexName': 'ShortlinksIndex',
-                    'KeySchema': [{
-                        'AttributeName': 'shortlinks_id', 'KeyType': 'HASH'
-                    }],
-                    'Projection': {
-                        'ProjectionType': 'INCLUDE', 'NonKeyAttributes': ['url']
-                    }
-                }
-            ],
+            },
+                                   {
+                                       'IndexName': 'ShortlinksIndex',
+                                       'KeySchema': [{
+                                           'AttributeName': 'shortlinks_id', 'KeyType': 'HASH'
+                                       }],
+                                       'Projection': {
+                                           'ProjectionType': 'INCLUDE', 'NonKeyAttributes': ['url']
+                                       }
+                                   }],
```

Comment:
Indentation issue. Unfortunately yapf has often troubles with complex structures (list with dictionary or dict with list, ...). Most of the time this can be fixed by simply adding trailing comma in list/dict. So here adding a comma after the last dict in the list would probably fixed the indentation from yapf.

Reply:
It did the trick. Thanks.